<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1155</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
      <item>
       <widget class="QGroupBox" name="menu_groupBox">
        <property name="title">
         <string>Главное меню</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="menu_verticalLayout">
           <item>
            <widget class="QPushButton" name="listpswdOpen_pushButton">
             <property name="text">
              <string>Менеджер паролей</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="settings_pushButton">
             <property name="text">
              <string>Настройки</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="pswdMg_groupBox">
        <property name="title">
         <string>Менеджер паролей</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
         </property>
         <item>
          <layout class="QHBoxLayout" name="pswdMg_horizontalLayout" stretch="2,0">
           <item>
            <layout class="QVBoxLayout" name="accountList_verticalLayout">
             <item>
              <layout class="QVBoxLayout" name="MgListAcc_verticalLayout">
               <item>
                <layout class="QHBoxLayout" name="findAcc_horizontalLayout" stretch="3,1">
                 <property name="sizeConstraint">
                  <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
                 </property>
                 <item>
                  <widget class="QLineEdit" name="findAcc_lineEdit"/>
                 </item>
                 <item>
                  <widget class="QPushButton" name="findAcc_pushButton">
                   <property name="text">
                    <string>найти</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Логин</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Пароль</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Сервис</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QScrollArea" name="scrollArea">
                 <property name="widgetResizable">
                  <bool>true</bool>
                 </property>
                 <widget class="QWidget" name="scrollAreaWidgetContents">
                  <property name="geometry">
                   <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>438</width>
                    <height>276</height>
                   </rect>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_3">
                   <item>
                    <layout class="QVBoxLayout" name="listAcc_verticalLayout">
                     <item>
                      <widget class="QPushButton" name="pushButton">
                       <property name="text">
                        <string>PushButton</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="exitPswdMg_pushButton">
               <property name="text">
                <string>Выйти</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="addNewAc_groupBox">
             <property name="title">
              <string>Добавить</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QVBoxLayout" name="addNewAc_verticalLayout">
                <item>
                 <layout class="QHBoxLayout" name="addNewAcServ_horizontalLayout">
                  <item>
                   <widget class="QLabel" name="addNewAcServ_label">
                    <property name="text">
                     <string>Сервис</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="addNewAcServ_lineEdit"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="addNewAcLog_horizontalLayout">
                  <item>
                   <widget class="QLabel" name="addNewAcLog_label">
                    <property name="text">
                     <string>Логин</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="addNewAcLog_lineEdit"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="addNewAcPswd_horizontalLayout">
                  <item>
                   <widget class="QLabel" name="addNewAcPswd_label">
                    <property name="text">
                     <string>Пароль</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="addNewAcPswd_lineEdit"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QPushButton" name="gemPswd_pushButton">
                  <property name="text">
                   <string>Сгенерировать пароль</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QGroupBox" name="gemPswd_groupBox">
                  <property name="title">
                   <string>Сгенерировать пароль</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_7">
                     <item>
                      <widget class="QCheckBox" name="gemPswdCap_checkBox">
                       <property name="text">
                        <string>заглавные</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QCheckBox" name="genPswdSym_checkBox">
                       <property name="text">
                        <string>символы</string>
                       </property>
                       <property name="checked">
                        <bool>false</bool>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QCheckBox" name="gemPswdLow_checkBox">
                       <property name="text">
                        <string>строчные</string>
                       </property>
                       <property name="checked">
                        <bool>true</bool>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QCheckBox" name="gemPswdNum_checkBox">
                       <property name="text">
                        <string>цифры</string>
                       </property>
                       <property name="checked">
                        <bool>true</bool>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="genPswd_pushButton">
                       <property name="text">
                        <string>Сгенерировать пароль</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <widget class="QSlider" name="horizontalSlider">
                     <property name="minimum">
                      <number>1</number>
                     </property>
                     <property name="maximum">
                      <number>128</number>
                     </property>
                     <property name="value">
                      <number>3</number>
                     </property>
                     <property name="orientation">
                      <enum>Qt::Orientation::Horizontal</enum>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="sliderValueLabel">
                     <property name="text">
                      <string>2</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="addNewAcButton_horizontalLayout">
                  <item>
                   <widget class="QPushButton" name="addNewAc_pushButton">
                    <property name="text">
                     <string>Сохранить</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="backFromNewAc_pushButton">
                    <property name="text">
                     <string>Отменить</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1155</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
